src/App.jsx

```diff
@@ -1,60 +1,178 @@
-import "./App.css"
+import { useState } from "react";
+import "./App.css";
 
 function App() {
+  const [panelOne, setPanelOne] = useState("0");
+  const [panelTwo, setPanelTwo] = useState("0");
+  const [panelMath, setPanelMath] = useState("+");
+  const [answer, setAnswer] = useState(0);
+
+  const addToPanelOne = (event) => {
+    console.log(event.currentTarget.value);
+    console.log("panel one is", panelOne);
+    if (panelOne === "0") {
+      setPanelOne(event.currentTarget.value);
+    } else if (event.currentTarget.value === "Clear") {
+      setPanelOne("0");
+    } else {
+      setPanelOne(panelOne.toString() + event.currentTarget.value.toString());
+    }
+  };
+
+  const addToPanelMath = (event) => {
+    console.log(event.currentTarget.value);
+    console.log("panel math is", panelMath);
+    setPanelMath(event.currentTarget.value);
+  };
+
+  const addToPanelTwo = (event) => {
+    console.log(event.currentTarget.value);
+    console.log("panel Two is", panelTwo);
+    if (panelTwo === "0") {
+      setPanelTwo(event.currentTarget.value);
+    } else if (event.currentTarget.value === "Clear") {
+      setPanelTwo("0");
+    } else {
+      setPanelTwo(panelTwo.toString() + event.currentTarget.value.toString());
+    }
+  };
+
+  const getAnswer = () => {
+    console.log(`calculating: ${panelOne} ${panelMath} ${panelTwo}`);
+    switch (panelMath) {
+      case "+":
+        setAnswer(Number(panelOne) + Number(panelTwo));
+        break;
+      case "-":
+        setAnswer(Number(panelOne) - Number(panelTwo));
+        break;
+      case "*":
+        setAnswer(Number(panelOne) * Number(panelTwo));
+        break;
+      case "÷":
+        setAnswer(Number(panelOne) / Number(panelTwo));
+        break;
+      default:
+        break;
+    }
+  };
+
+  function resetCalculator() {
+    setPanelOne("0");
+    setPanelTwo("0");
+    setPanelMath("+");
+  }
+
+  function calculateAndReset() {
+    getAnswer();
+    resetCalculator();
+  }
 
   return (
     <div className="calculator">
       <div className="panel">
-        <p>0</p>
+        <p>{panelOne}</p>
         <div className="numbers">
-          <button>1</button>
-          <button>2</button>
-          <button>3</button>
-          <button>4</button>
-          <button>5</button>
-          <button>6</button>
-          <button>7</button>
-          <button>8</button>
-          <button>9</button>
-          <button>0</button>
-          <button>Clear</button>
+          <button value={1} onClick={addToPanelOne}>
```

Comment:
this could be simplified to `<Button onClick={() => updateLeftNumber("1")} > ... ` so you avoid having to parse / use the `event` object.